libs/core/src/lib/input-group/input-group.component.scss

```diff
@@ -7,11 +7,10 @@ fd-input-group {
         max-height: 100%;
         height: 100%;
     }
-}
-.fd-input-group {
-    .fd-input-group__addon--button > * {
-        border-color: var(--fd-forms-border-color, #89919a);
-        height: 100%;
-        max-height: 100%;
+
+    .fd-input-group__addon--button .fd-input-group__button {
+        &:hover {
```

Comment:
Should this be in styles?

Reply:
It is on styles, but on NGX we have some issues with classes hierarchy, so it's mandatory to add it there aswell.